.github/ISSUE_TEMPLATE/zpa_epic.md

```diff
@@ -0,0 +1,80 @@
+---
+
+name: Zeebe Process Automation team Epic
+about: Prepare a new Epic for the ZPA team
+title: ''
+labels: 'kind/epic'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+
+
+## Introduction
+<!-- Provide a short summary about the problem this Epic will solve -->
+
+
+## Concept
+<!-- Provide a high-level explanation of the concept. For example, when a new BPMN element gets added,
+explain how it is supposed to work. You can add relevant pieces of the specification here as well -->
```

Comment:
```suggestion
## Concept
<!-- Provide a high-level explanation of the implementation concept. For example, when a new BPMN element gets added,
explain how it is supposed to work. -->
```

Previously, I've used the concept to explain the implementation concept rather than the concept of the feature itself (i.e. from the user's perspective).